docs/participant/pages/04_details.md

```diff
@@ -0,0 +1,21 @@
+# Details
+
+### About this page:
+
+Users fill out "participant cards" (a repeatable fieldset) to collect detailed information about participants who are recertifying. This info will both help staff find the participant in M-SPIRIT and ensure we request the right kinds of documents. Because WIC participants do not know their household ID and using EBT # would require additional steps for staff, we will rely on the participant's name and date of birth as the data points staff can use to look up the participant in M-SPIRIT. We ask adjunctive eligibility per participant since it's possible one participant has it and another doesn't.
+
+### Considerations for other states:
+
+If your state is able to surface who is recertifying, for example by using MIS data, then you can avoid this and the /count pages all together and instead surface to the user who is recertifying.
```

Comment:
Same comment as for count. Suggesting dropping this.

```suggestion
If your state is able to surface who is recertifying, for example by using MIS data, then you can avoid this and the /count pages all together and instead surface to the user who is recertifying.
```

Please add something like this:
```
- Replace the adjunctive eligibility question with the programs that qualify a participant for WIC adjunctive eligibility in your state
```